Tests/SwiftSyntaxMacroExpansionTest/PeerMacroTests.swift

```diff
@@ -62,9 +60,10 @@ final class PeerMacroTests: XCTestCase {
                 "add 'async'"
               ),
               changes: [
-                FixIt.Change.replace(
-                  oldNode: Syntax(funcDecl.signature),
-                  newNode: Syntax(newSignature)
+                FixIt.Change(
+                  parent: funcDecl,
+                  replacingChildAt: \.signature.effectSpecifiers,
+                  with: newEffects
```

Comment:
I guess `FunctionDeclSyntax.signature.effectSpecifiers` is a valid use case of this new API?